package.json

```diff
@@ -9,9 +9,10 @@
   },
   "dependencies": {
     "core-js": "^3.6.5",
+    "nanoid": "^3.1.23",
     "vue": "^2.6.11",
-    "vue-stash": "^2.0.1-beta",
-    "vue-router": "^3.2.0"
+    "vue-router": "^3.2.0",
+    "vue-stash": "^2.0.1-beta"
```

Comment:
Should revert this change since it's not related to this ticket.

Reply:
Updated!